bayesian/utils.py

```diff
@@ -925,3 +927,12 @@ def generate_content_hash(content):
     """Return the sha1 digest of a string."""
     hash_object = hashlib.sha1(content.encode('utf-8'))
     return hash_object.hexdigest()
+
+
+@lru_cache_function(max_size=2048, expiration=60 * 60 * 24)
+def get_core_dependencies(runtime):
+    """Return core dependencies for each runtime."""
+    fetched_file = fetch_file_from_github(CORE_DEPENDENCIES_REPO_URL, 'core.json')
+    dependencies = fetched_file[0].get('content', {})
```

Comment:
`fetched_file_from_github` returns the data in the raw format you need to parse it in JSON format.